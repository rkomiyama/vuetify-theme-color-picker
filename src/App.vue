<template>
  <v-app>
    <MainSiteContainer />
  </v-app>
</template>

<script>
import MainSiteContainer from '@/components/MainSiteContainer'

export default {
  name: 'App',
  components: {
    MainSiteContainer
  }
};
</script>
