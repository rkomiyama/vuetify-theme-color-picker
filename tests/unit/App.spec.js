import Vue from 'vue';
import Vuetify from 'vuetify';
import { shallowMount } from '@vue/test-utils'
import { createRenderer } from 'vue-server-renderer';
import App from '@/App.vue'

Vue.use(Vuetify)
let wrapper

describe('App.vue', () => {
  beforeEach(() => {
    wrapper = shallowMount(App)
  })

  afterEach(() => {
    wrapper.destroy()
  })

  it('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy();
  })

  it('App renders the same HTML', () => {
    const renderer = createRenderer();
    renderer.renderToString(wrapper.vm, (err, str) => {
    if (err) throw new Error(err);
      expect(str).toMatchSnapshot();
    });
  });
})
