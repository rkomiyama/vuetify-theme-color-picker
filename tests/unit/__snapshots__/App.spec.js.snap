// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue App renders the same HTML 1`] = `
<v-app-stub id="app">
  <MainSiteContainer-stub></MainSiteContainer-stub>
</v-app-stub>
`;
